### ISSUE TITLE
bbcode for forum post

### ISSUE THREAD

**Issue by cosme12** [delight-im/OpenSoccer#292]
Is it possible to add some simple bbcode tags to the forums? Maybe not images or url but bold, italic and some color. I think it will make posts more clear and not just a wall of text.

**Comment by mwaclawek** [comment]
Didn't think about this yet. But what about `[b]`, `[i]`, `[u]` and `[s]` (struck-through) only? Nore sure about colors or font sizes.

**Comment by cosme12** [comment]
Yes! That would be more than enough and a huge improvement without big changes.
